packages/mcp/src/api/router.ts

```
@@ -0,0 +1,363 @@
+        memories: z.array(z.any()),
+        config: z
+          .object({
+            complexityLevel: z.enum(['basic', 'standard', 'detailed']).optional(),
+            includeTrajectory: z.boolean().optional(),
+            maxDescriptors: z.number().optional(),
+          })
+          .optional(),
+      })
+    )
+    .output(z.any())
+    .mutation(async ({ input }: { input: any }) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)